pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -14,16 +14,21 @@ import (
 )
 
 const (
-	trivyIgnore = ".trivyignore"
+	defaultTrivyIgnore = ".trivyignore"
 )
 
 var (
 	sources = []string{Nvd, RedHat, Debian, DebianOVAL, Alpine,
 		RubySec, RustSec, PhpSecurityAdvisories, NodejsSecurityWg, PythonSafetyDB}
 )
 
-func FillAndFilter(vulns []DetectedVulnerability, severities []Severity, ignoreUnfixed bool) []DetectedVulnerability {
-	ignoredIDs := getIgnoredIDs()
+func FillAndFilter(vulns []DetectedVulnerability, severities []Severity, ignoreUnfixed bool, trivyIgnore string) []DetectedVulnerability {
+	var ignoredIDs []string
```

Comment:
```suggestion
	ignoredIDs := getIgnoredIDs(ignoreFile)
```

Reply:
I'm sorry that my explanation was inadequate. `trivyIgnore` is not clear (I know this variable was defined by me), so I suggested that we should change the variable name to `ignoreFile`.

Reply:
@knqyf263 
I misunderstood your review, so sorry
I fix variable name to "ignoreFile" from "trivyignore" to more clear